src/agent.py

```
@@ -177,7 +306,282 @@ async def chat(
+        chat_history: The chat history
+        session_id: The session ID
+        
+    Returns:
+        The ToM inference
+    """
+    # Run ToM inference
+    print(f"[TOM] Running ToM inference for session {session_id}")
+    tom_start_time = asyncio.get_event_loop().time()
+    
+    # Get chat history length to determine if this is a new conversation
+    tom_inference_response = await get_tom_inference(
```

Ruff (0.8.2)

Undefined name `get_tom_inference`

(F821)